### ISSUE TITLE
BaseButton's onPress not working if inside TapGestureHandler on ios

### ISSUE THREAD

**Issue by ghost** [software-mansion/react-native-gesture-handler#938]
Hi, on iOS the BaseButton's onPress is not triggering when nested inside a TapGestureHandler.
Problem appears only on iOS, on Android it works

```
<TapGestureHandler onHandlerStateChange={onCloseState}>
          <Animated.View
            style={{
              ...styles.closeButton,
              transform: [{ rotate: concat(roteteCross, 'deg') }]
            }}
          >
            <BaseButton onPress={() => Keyboard.dismiss()}>
              <Icon type="material" name="close" />
            </BaseButton>
          </Animated.View>
</TapGestureHandler>
```

**Comment by alexichristakis** [comment]
Also facing this issue

**Comment by vance-liu** [comment]
Same problem

**Comment by uujx** [comment]
Same here. And how do I handle a tap with animation and in the meantime with some js logic? Look like I have to give an Animated.event to onHandlerStateChange and there is no place for anything else like setState.

**Comment by nicocolascrete** [comment]
Same trouble here. Did you have any solution?

**Comment by tomekzaw** [comment]
May be related to #594.

**Comment by j-piasecki** [comment]
Hi! This should be fixed by setting `cancelsTouchesInView` prop on `TapGestureHandler` to `false`. This prop was added in https://github.com/software-mansion/react-native-gesture-handler/pull/1688 which will be included in the version 2.4.0 of Gesture Handler.

I will close this issue for now, and if the new prop doesn't solve your problem feel free to reopen it.

**Comment by fukemy** [comment]
cancelsTouchesInView not working to me
